components/causal_ts/src/tso.rs

```diff
@@ -225,7 +331,7 @@ impl<C: PdClient + 'static> BatchTsoProvider<C> {
 
         // Duration::ZERO means never renew automatically. For test purpose ONLY.
         if self.renew_interval > Duration::ZERO {
-            self.renew_worker
+            self.periodical_renew_worker
```

Comment:
why not let it handle `renew_thread` as well

Reply:
Fixed. PTAL~